src/video-player/components/runtime.tsx

```diff
@@ -57,55 +57,57 @@ export const Runtime: React.FC<IProps> = ({ authoredState, interactiveState, set
   viewedTimestampRef.current = viewedTimestamp;
 
   useEffect(() => {
-    const player: videojs.Player = videojs(playerRef.current,
-      {
-        controls: true,
-        fluid: !(authoredState.fixedAspectRatio || authoredState.fixedHeight),
-        // This is a new property not supported by the current types
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-ignore
-        crossOrigin: "anonymous"
-      }, () => {
-        const url = authoredState.videoUrl ? authoredState.videoUrl : "";
-        player.src(url);
-
-        if (authoredState.captionUrl && player.textTracks().length === 0) {
-          player.addRemoteTextTrack({
-            kind: "captions",
-            language: "en",
-            label: "English",
-            src: authoredState.captionUrl,
-            default: captionsOnByDefault
-          }, false);
-          // Store a ref to the captions track s we can check visibility later. There is no easy toggle event for captions show/hide
-          captionsTrackRef.current = player.textTracks()[0];
-          if (captionsTrackRef.current) {
-            captionsTrackRef.current.addEventListener("cuechange", () => {
-              const currentCue = captionsTrackRef.current?.activeCues?.length && captionsTrackRef.current?.activeCues.length > 0 && captionsTrackRef.current?.activeCues[0];
-              log("cue change", { videoUrl: authoredState.videoUrl, currentCue: (currentCue as VTTCue).text });
-            });
+    if (playerRef.current) {
+      const player: videojs.Player = videojs(playerRef.current,
+        {
+          controls: true,
+          fluid: !(authoredState.fixedAspectRatio || authoredState.fixedHeight),
+          // This is a new property not supported by the current types
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
```

Comment:
This is not new code, so feel free to ignore this, but if you use `@ts-expect-error` here then it'll notify when the types are updated to support the new property.